packages/worker/src/interceptors.ts

```diff
@@ -85,6 +86,68 @@ export interface ActivityInboundCallsInterceptorFactory {
   (ctx: ActivityContext): ActivityInboundCallsInterceptor;
 }
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Nexus Interceptors
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type NexusInterceptorsFactory = (ctx: nexus.OperationContext) => NexusInterceptors;
+
+/**
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export type NexusInterceptors = {
+  inbound?: NexusInboundCallsInterceptor;
+  outbound?: NexusOutboundCallsInterceptor;
+};
+
+/**
+ * A function that takes a Nexus Context and returns an interceptor
```

Comment:
I don't think I understand this comment

Reply:
Improved that specific typedoc, as well as that of the equivalent `ActivityInterceptorsFactory` type above.

However, I'd point out that typedocs on everything interceptors-related (Client/Workflow/Activity/Nexus) is very disappointing. I'm taking note to revisit _all_ interceptors soon.